### PR TITLE
Implement availability date restrictions update

### PR DESCRIPTION
**Pull request by DFSko**

### CODE REVIEW

**Review by oleg-n-k — APPROVED** [review]

File: HappyTravel.Komoro.JuniperChannelManager/HappyTravel.Komoro.JuniperChannelManager.csproj [hunk 1]
```diff
@@ -16,6 +16,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\HappyTravel.Komoro.Common\HappyTravel.Komoro.Common.csproj" />
+    <ProjectReference Include="..\HappyTravel.Komoro.Data\HappyTravel.Komoro.Data.csproj" />
```
Comment (AlexeyShok) [inline HappyTravel.Komoro.JuniperChannelManager/HappyTravel.Komoro.JuniperChannelManager.csproj:19]: I think that the channel manager should not work directly with komoro data. You need to use Api service methods to change the data. Working directly with data from the channel manager greatly complicates code cohesion.

Comment (AlexeyShok) [reply HappyTravel.Komoro.JuniperChannelManager/HappyTravel.Komoro.JuniperChannelManager.csproj:19]: _

**Review by AlexeyShok — CHANGES_REQUESTED** [review]

File: HappyTravel.Komoro.JuniperChannelManager/HappyTravel.Komoro.JuniperChannelManager.csproj (hunk 1 above)
Comment (AlexeyShok) [reply HappyTravel.Komoro.JuniperChannelManager/HappyTravel.Komoro.JuniperChannelManager.csproj:19]: _

**Review by AlexeyShok — CHANGES_REQUESTED** [review]

File: HappyTravel.Komoro.JuniperChannelManager/HappyTravel.Komoro.JuniperChannelManager.csproj (hunk 1 above)
Comment (DFSko) [reply HappyTravel.Komoro.JuniperChannelManager/HappyTravel.Komoro.JuniperChannelManager.csproj]: Fixed

**Review by AlexeyShok — APPROVED** [review]

**Pull request merged**